### ISSUE TITLE
Include screenshots in the README

### ISSUE THREAD

**Issue by ZelimDamian** [google/flatui#4]
It is extremely important to include examples of GUI use for those having the first look. Good example is ImGUI https://github.com/ocornut/imgui.

**Comment by gwvo** [comment]
Agreed. Though, this is a 0.9 release, and we haven't really made a good looking demo yet. The other issue is that unlike ocornut's library, FlatUI does very little in dictating the looks, that really depends on the user.

**Comment by BlackRookSix** [comment]
@gwvo - The fact that the library does not dictate looks in and of itself is a key item that should be explained in the README or the Documentation.  Examples, screenshots, even some sort of a live, editable demo would be a huge boon to the understanding of new users coming to the project.

**Comment by aardappel** [comment]
Yup, we realize we need to do a lot better in that regard.

**Comment by onqtam** [comment]
no longer version 0.x - I'd really like some screenshots before even trying to build it :/

**Comment by GuacheSuede** [comment]
Do you think we may still have screenshots ?

**Comment by carlfindahl** [comment]
Hi, I am going to jump into this issue and **remind you** that this is a GUI library, yet you have absolutely no screenshots of UI's created with it, even after all this time.

I would appreciate it a lot if you could point us to some references or add some examples of what is possible to do with this on your wiki/readme, as otherwise, I'm not going to ever put in the effort to use it.

It will scare so many users away when they can't even see a single minimal example of what you can do with this. Thanks.

**Comment by aardappel** [comment]
@carlfindahl I'm sad to say that this project currently doesn't have an active maintainer.. so unless someone wants to pick it up, the screenshots may not happen.
